transcode/transcode.go

```diff
@@ -212,36 +213,59 @@ func RunTranscodeProcess(transcodeRequest TranscodeSegmentRequest, streamName st
 				break
 			}
 
-			// d. Transmux the single .ts file into an .mp4 file
-			mp4OutputFileName := concatTsFileName[:len(concatTsFileName)-len(filepath.Ext(concatTsFileName))] + ".mp4"
-			err = video.MuxTStoMP4(concatTsFileName, mp4OutputFileName)
-			if err != nil {
-				log.Log(transcodeRequest.RequestID, "error transmuxing", "err", err)
-				continue
-			}
-
-			// e. Upload mp4 output file
-			mp4OutputFile, err := os.Open(mp4OutputFileName)
-			defer os.Remove(mp4OutputFileName)
-			if err != nil {
-				log.Log(transcodeRequest.RequestID, "error opening mp4", "file", mp4OutputFileName, "err", err)
-				break
+			// d. Mux the .ts file to generate either a regular MP4 (w/ faststart) or fMP4 packaged with HLS/DASH
+			var mp4OutputFiles []string
+			if !transcodeRequest.Mp4Fragmented {
```

Comment:
Could we flip this so that we're not checking for a negative? Or like in my other comment, maybe support both?

Reply:
Yeah, will update this.